rollup.config.js

```diff
@@ -93,7 +96,13 @@ async function main()
         // Check for bundle folder
         const external = Object.keys(pkg.dependencies || []);
         const basePath = path.relative(__dirname, pkg.location);
-        const input = path.join(basePath, 'src/index.js');
+        let input = path.join(basePath, 'src/index.js');
```

Comment:
Could you flip this and check for the Ts file first and fallback to Js? That way we can simply remove the existsSync check when we’re are converted. Also, could you add a TODO comment here to make sure we remove it later?